docs/guides/chip_tool_guide.md

```diff
@@ -778,25 +849,104 @@ following command:
+The SSID `wifi_test` and the password `admin123` is the required information about the Wi-Fi network to which the device is going to be commissioned.
+The onboarding dataset payload `34970112332` (short manual pairing code) is used to discover and commission the device.
+
+```
+$ ./chip-tool pairing code-wifi 1 wifi_test admin123 34970112332
+```
+
+The following command commissions the device with the node ID `1` to the Matter fabric.
+The onboarding dataset payload `MT:8IXS142C00KA0648G00` (QR code payload) is used to discover and commission the device.
+
+```
+$ ./chip-tool pairing code 1 MT:8IXS142C00KA0648G00
+```
+
+#### Attestation-related flags
+
+The `pairing` commissioning command can be run with several flags that allow you
+to modify attestation-related settings:
+
+-   `--paa-trust-store-path` - Use to provide the path to the directory that
+    contains the information about Product Attestation Authority (PAA)
+    certificates. The path can be absolute or relative to the current working
+    directory. With this flag, the CHIP Tool looks for the PAA certificate that
+    matches the PAI and the DAC certificates programmed on the device. Without
+    this flag, the CHIP Tool uses the built-in test PAA certificate.
+
+-   `--cd-trust-store-path` - Use to provide the path to the directory containing the key that
+    is used to validate the device's Certification Declaration. The path can be absolute or
+    relative to the current working directory. With this flag, the CHIP tool
+    looks for the public key that successfully validate device Certification Declaration's signature.
+    Without this flag, the CHIP Tool uses well-known built-in public keys 
+    (built-in public keys `src/credentials/attestation_verifier/DefaultDeviceAttestationVerifier.cpp`)
+
+
+-   `--only-allow-trusted-cd-keys` - Use to only allow the keys from `--cd-trust-store-path`
+    and not the built-in test key. If the flag is not provided or it is provided with the
+    value `false`, untrusted CD verifying keys are allowed. If it is
+    provided with the value `true` (`--only-allow-trusted-cd-keys true`), test
+    keys are disallowed and CD signed with the test key will not be accepted.
+
+-   `--bypass-attestation-verifier` - Use to bypass the attestation verifier. If
+    the flag is not provided or it is provided with the value `false`, the
+    attestation verifier is not bypassed. If it is provided with the value `true`
+    (`--bypass-attestation-verifier true`), the commissioning will continue
+    in case of the attestation verification failure. The failure can be caused
+    by errors in Certification Declaration, PAA or PAI certificates, or
+    in the Device Attestation Certificate. This option can be helpful if you want
+    to quickly commission a device with PAI and DAC certificates based on
+    not known PAA and/or with not known Certification Declaration public key.
```

Comment:
A Certification Declaration does not have a "public key", so this is very confusing.  How about:
```suggestion
    an unknown PAA and/or with a Certification Declaration signed by an unknown key.
```